Analysis/HiggsTauTau/scripts/runAllCats.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Could this script be put in your subdir of scripts/ which you made to keep all optimisation stuff together, or is it intentionally kept separate?